pytorch_lightning/utilities/distributed.py

```diff
@@ -216,7 +221,7 @@ def backward(ctx, *grad_output):
 
 
 def all_gather_ddp_if_available(
-    tensor: Union[torch.Tensor], group: Optional[Any] = None, sync_grads: bool = False
+    tensor: torch.Tensor, group: Optional[torch.distributed.ProcessGroup] = None, sync_grads: bool = False
```

Comment:
`torch.distributed.ProcessGroup` is not available for PyTorch <1.7 on Windows

I think writing it in quotes should work

Reply:
Thanks, looks it worked :)